.env.template

```diff
@@ -0,0 +1,9 @@
+DEAD_MOROZ_DATABASE_USERNAME=DEAD_MOROZ_DATABASE_USERNAME
+DEAD_MOROZ_DATABASE_PASSWORD=DEAD_MOROZ_DATABASE_PASSWORD
+
+MAILER_SENDER=MAILER_SENDER
+
+REDIS_URL=REDIS_URL
+
+# To force chrome driver during tests:
+# FORCE_CHROME=true
```

Comment:
You can also store some default values here so the potential new developer can just do `cp .env.template .env` and will have a working environment (or at least almost working). Something like:
```bash
DEAD_MOROZ_DATABASE_USERNAME=dead_moroz
DEAD_MOROZ_DATABASE_PASSWORD=password

MAILER_SENDER=foo@gmail.com

REDIS_URL=redis://localhost:6379/0

# To force chrome driver during tests:
# FORCE_CHROME=true
```